kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/preprocess/SinkRecordFilterTest.java

```diff
@@ -0,0 +1,198 @@
+package com.wepay.kafka.connect.bigquery.preprocess;
+
+import com.jayway.jsonpath.InvalidPathException;
+import com.wepay.kafka.connect.bigquery.SinkConnectorPropertiesFactory;
+import com.wepay.kafka.connect.bigquery.config.BigQuerySinkConfig;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Logger;
```

Comment:
Should use `org.slf4j.Logger` and `org.slf4j.LoggerFactory`

Reply:
Done